packages/fuels/src/cli.ts

```diff
@@ -32,7 +32,7 @@ export const configureCli = () => {
 
   program.version(versions.FUELS, '-v, --version', 'Output the version number');
   program.helpOption('-h, --help', 'Display help');
-  program.addHelpCommand('help [command]', 'Display help for command');
+  program.helpCommand('help [command]', 'Display help for command');
```

Comment:
Not directly related to this PR, but there's another instance of this [here](https://github.com/FuelLabs/fuels-ts/blob/aa/feat/support-deploying-proxy-contracts/packages/create-fuels/src/lib/setupProgram.ts#L31)

Reply:
Yes, but that one requires a slightly different implementation since it calls `.addHelpCommand()` with no parameters.

I would be OK with adding this to this PR or another one.

> cc @Dhaiwat10

Reply:
It's not directly related so a follow up PR should be fine

Reply:
- https://github.com/FuelLabs/fuels-ts/issues/3207